railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -5,6 +5,11 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <meta name="apple-mobile-web-app-capable" content="yes">
     <meta name="mobile-web-app-capable" content="yes">
+    <%- unless options[:skip_hotwire] || options[:skip_javascript] -%>
+    <meta name="view-transition" content="same-origin">
+    <meta name="turbo-refresh-method" content="morph">
+    <meta name="turbo-refresh-scroll" content="preserve">
```

Comment:
Morphing new `<body>` elements instead of replacing requires additional considerations. Using it by default presents some surprises for both developers and end-users.

Morphing is somewhat incompatible with custom elements or other patterns that rely on maintaining client-side through HTML elements and attributes.

For example, Morph-enabled Page Refreshes will break Action Text-rendered `<trix-editor>` elements without explicitly rendering them within an element marked with `[data-turbo-permanent]`.

* https://github.com/hotwired/turbo/issues/1083
* https://github.com/hotwired/turbo/issues/1210
* https://github.com/hotwired/turbo/issues/1239
* https://github.com/hotwired/turbo-rails/issues/533
* https://github.com/hotwired/turbo-rails/pull/688

Turbo provides a variety of integration points to respond to these scenarios, but there are not one-size fits all solutions. I don't think Morph-enabeld rendering is an appropriate default.